building/build-debs/homeworld-admin-tools/src/verify.py

```diff
@@ -75,7 +75,7 @@ def check_keygateway():
     try:
         access.call_keyreq("check")
     except subprocess.CalledProcessError as e:
-        command.fail("keygateway check failed: %s" % e)
+        command.fail("keygateway check failed (do you have valid Kerberos tickets?): %s" % e)
```

Comment:
It might be worth detecting the exact failure mode ('returned non-zero exit status 1') and only providing the extra information in that case, and then also being louder about the the error, rather than sticking it in the middle of a longer error message.

Maybe an error like:

    keygateway check failed: [...]
    ** do you have valid kerberos tickets? **